web/src/components/TtsPlayer.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { useTheme } from 'styled-components'
+
+import { CloseIcon, PauseIcon, PlaybackIcon, PlayIcon } from '../assets'
+import dimensions from '../constants/dimensions'
+import TtsHelpModal from './TtsHelpModal'
+import Button from './base/Button'
+import Icon from './base/Icon'
+
+const StyledTtsPlayer = styled.dialog<{ $isPlaying: boolean }>`
+  background-color: ${props => props.theme.colors.ttsPlayerBackground};
+  border-radius: 28px;
+  width: 388px;
+  display: flex;
+  flex-direction: ${props => (props.$isPlaying ? 'column' : 'row')};
+  justify-content: center;
+  align-items: center;
+  padding: 8px;
+  position: sticky;
+  bottom: 4px;
+  min-height: 92px;
+  gap: ${props => (props.$isPlaying ? '4px;' : '12px')};
+  border-color: transparent;
+
+  @media ${dimensions.smallViewport} {
+    width: 88%;
+  }
+`
+
+const verticalMargin = 12
+
+const StyledPanel = styled.div<{ $isPlaying?: boolean }>`
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  margin: ${props => (props.$isPlaying ? verticalMargin : 0)}px 0;
+`
+
+const BaseButton = styled(Button)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition:
+    box-shadow 0.2s ease,
+    transform 0.1s ease;
+
+  &:active {
+    box-shadow: none;
+    transform: translateY(2px);
+  }
+`
+
+const StyledPlayIcon = styled(BaseButton)`
+  background-color: ${props => props.theme.colors.ttsPlayerPlayIconColor};
+  width: 48px;
+  height: 48px;
+  border-radius: 48px;
+  box-shadow: 1px 4px 8px 1px grey;
+`
+
+const StyledBackForthButton = styled(Button)`
+  display: flex;
+  flex-direction: row;
+  gap: 4px;
+  align-items: flex-end;
+`
+
+const PlayButtonIcon = styled(Icon)`
+  color: ${props => props.theme.colors.ttsPlayerBackground};
+`
+
+const BackForthIcon = styled(Icon)<{ $flip: boolean }>`
```

Comment:
:wrench: We have this functionality implemented. Please use the prop `reverse` instead.